integration_tests/web/test_conversations_connect.py

```diff
@@ -0,0 +1,140 @@
+import logging
+import os
+import time
+from typing import Optional
+import unittest
+
+from slack_sdk.web.slack_response import SlackResponse
+from slack_sdk.errors import SlackApiError
+from integration_tests.env_variable_names import (
+    SLACK_SDK_TEST_SENDER_BOT_TOKEN,
+    SLACK_SDK_TEST_RECEIVER_BOT_TOKEN,
+    SLACK_SDK_TEST_RECEIVER_BOT_USER_ID,
+)
+from integration_tests.helpers import async_test
+from slack_sdk.web import WebClient
+from slack_sdk.web.async_client import AsyncWebClient
+
+
+class TestWebClient(unittest.TestCase):
+    """Runs integration tests with Slack API for conversations.* endpoints
+    To run, we use two workspace-level bot tokens,
+    one for the inviting workspace(list and send invites) another for the recipient
+    workspace (accept and approve) sent invites. Before being able to run this test suite,
+    we also need to have manually created a slack connect shared channel and added
+    these two bots as members first. See: https://api.slack.com/apis/connect
+
+    In addition to conversations.connect:* scopes, your sender bot token should have channels:manage scopes.
+    """
+
+    def setUp(self):
+        self.logger = logging.getLogger(__name__)
+        self.sender_bot_token = os.environ[SLACK_SDK_TEST_SENDER_BOT_TOKEN]
+        self.receiver_bot_token = os.environ[SLACK_SDK_TEST_RECEIVER_BOT_TOKEN]
+        self.sender_sync_client: WebClient = WebClient(token=self.sender_bot_token)
+        self.sender_async_client: AsyncWebClient = AsyncWebClient(
+            token=self.sender_bot_token
+        )
+        self.receiver_sync_client: WebClient = WebClient(token=self.receiver_bot_token)
+        self.receiver_async_client: AsyncWebClient = AsyncWebClient(
+            token=self.receiver_bot_token
+        )
+
+    def tearDown(self):
+        pass
+
+    def test_sync(self):
+        sender = self.sender_sync_client
+        receiver = self.receiver_sync_client
+        channel_id: Optional[str]
```

Comment:
If there is no assignment and we access the value in finally clause, it should raise an exception.
```suggestion
        channel_id: Optional[str] = None
```